### PR TITLE
Added "" for the version at line 25

### PR DESCRIPTION
**Pull request by nougator**

### CODE REVIEW

File: src/getting-started/starter-templates.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ crate-type = ["cdylib"]
 
 [dependencies]
 # for web_sys
-yew = 0.15
+yew = "0.15"
```
Comment (teymour-aldridge) [inline src/getting-started/starter-templates.md]: Would you mind bumping up the version to the latest version of Yew?
```suggestion
yew = "0.16"
```

Comment (nougator) [reply src/getting-started/starter-templates.md:25]: Nope

Comment (teymour-aldridge) [reply src/getting-started/starter-templates.md]: Thanks (I assume you mean no to not minding, rather than no to updating the version)!

Comment (nougator) [reply src/getting-started/starter-templates.md:25]: Yes right

**Pull request merged**